src/SQLite3-Core/SQLite3BaseConnection.class.st

```diff
@@ -1,5 +1,7 @@
 "
 I represent a connection to an SQLite database. I provide a thin wrapper over the SQLite C API.
+
+Do not use this class directory, please check and use my subclass SQLite3Connection
```

Comment:
```suggestion
Do not use this class directly, please check and use my subclass SQLite3Connection
```

Reply:
Fixed in https://github.com/pharo-rdbms/Pharo-SQLite3/pull/49/commits/71a491174ea37068c828a73d07e9b9013165a8a3